Renv/.rstudio_config_dir/rstudio-prefs.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I'm not too familiar with specifying the settings here, but would this be considered a pretty standard configuration or an opinionated one? Is this file automatically updated by RStudio?

Reply:
To be clear this is not applied by default, the RStudio session will open vanilla. If you uncomment the line in the Dockerfile that copies this file to ~/.config/rstudio then these prefs will apply. They are my moderately opinionated defaults, dark mode, tabs instead of spaces, visible whitespace, document outline open by default, extra source column, plus a couple of other things. I was going for 'look how much you can customise the experience' to showcase some of the useful lesser known settings in my example of a custom config but I get it if you think it'd be better to just have this apply vanilla settings if you 'enable' it by uncommenting the line in the Dockerfil. I can make vanilla the default custom config and include the more custom example with a different filename if you like? I should also expand the README, and link to this package for customising your prefs come to think of it:
https://www.danieldsjoberg.com/rstudio.prefs/index.html

If the user changes their RStudio preferences in the GUI then copies the file from ~/.config/rstudio to .rstudio_config_dir and uncomments the line then their defaults would apply going forward. As it is I think preferences are lost between sessions.